SPECS/kata-containers-cc/drop-mut-for-variables-tahat-are-not-mutated.patch

```diff
@@ -0,0 +1,54 @@
+From a17efe9e87d691bc4c0b7f3ef503096993f3a9d6 Mon Sep 17 00:00:00 2001
+From: Muhammad Falak R Wani <falakreyaz@gmail.com>
+Date: Thu, 14 Sep 2023 16:10:09 +0530
+Subject: [PATCH 1/2] libs: kata-types: drop mut for vars that are immutable
+
+Signed-off-by: Muhammad Falak R Wani <falakreyaz@gmail.com>
+---
+ src/libs/kata-types/src/annotations/mod.rs | 4 ++--
+ 1 file changed, 2 insertions(+), 2 deletions(-)
+
+diff --git a/src/libs/kata-types/src/annotations/mod.rs b/src/libs/kata-types/src/annotations/mod.rs
+index 3af0563..db4e9f7 100644
+--- a/src/libs/kata-types/src/annotations/mod.rs
++++ b/src/libs/kata-types/src/annotations/mod.rs
+@@ -470,8 +470,8 @@ impl Annotation {
+         let u32_err = io::Error::new(io::ErrorKind::InvalidData, "parse u32 error".to_string());
+         let u64_err = io::Error::new(io::ErrorKind::InvalidData, "parse u64 error".to_string());
+         let i32_err = io::Error::new(io::ErrorKind::InvalidData, "parse i32 error".to_string());
+-        let mut hv = config.hypervisor.get_mut(hypervisor_name).unwrap();
+-        let mut ag = config.agent.get_mut(agent_name).unwrap();
++        let hv = config.hypervisor.get_mut(hypervisor_name).unwrap();
```

Comment:
How do we know this is safe?

Reply:
My understanding is these makes it more restrictive, i.e., we remove the mutable property of these values, sort of like making them `const` by default.